ui/tests/integration/components/search-select-test.js

```diff
@@ -0,0 +1,175 @@
+import { module, test, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { create } from 'ember-cli-page-object';
+import { typeInSearch, clickTrigger } from 'ember-power-select/test-support/helpers';
+import Service from '@ember/service';
+import { render } from '@ember/test-helpers';
+import { run } from '@ember/runloop';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+import searchSelect from '../../pages/components/search-select';
+
+const component = create(searchSelect);
+
+const storeService = Service.extend({
+  query(modelType) {
+    return new Promise((resolve, reject) => {
+      switch (modelType) {
+        case 'policy/acl':
+          resolve([{ id: '1', name: '1' }, { id: '2', name: '2' }, { id: '3', name: '3' }]);
+          break;
+        case 'policy/rgp':
+          reject({ httpStatus: 403, message: 'permission denied' });
+          break;
+        case 'identity/entity':
+          resolve([{ id: '7', name: 'seven' }, { id: '8', name: 'eight' }, { id: '9', name: 'nine' }]);
+          break;
+        case 'server/error':
+          var error = new Error('internal server error');
+          error.httpStatus = 500;
+          reject(error);
+          break;
+        default:
+          reject({ httpStatus: 404, message: 'not found' });
+          break;
+      }
+      reject({ httpStatus: 404, message: 'not found' });
+    });
+  },
+});
+
+module('Integration | Component | search select', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    run(() => {
+      this.owner.unregister('service:store');
+      this.owner.register('service:store', storeService);
+    });
+  });
+
+  test('it renders', async function(assert) {
+    const models = ['policy/acl'];
+    this.set('models', models);
+    this.set('onChange', sinon.spy());
+    await render(hbs`{{search-select label="foo" models=models onChange=onChange}}`);
```

Comment:
Smallest nit (sorry I missed this before) - could you change this to use angle-bracket syntax? Just think it'd be good to use it on new usage of components (even in tests).